packages/hawtio/src/plugins/shared/tree/node.ts

```diff
@@ -25,6 +25,10 @@ export interface OptimisedMBeanInfo extends Omit<MBeanInfo, 'attr' | 'op'> {
   canInvoke?: boolean
 }
 
+export interface ErrorMBeanInfo {
+  error: string
+}
```

Comment:
`ErrorResponse` type is already defined at jolokia.js. We should use it instead.
https://github.com/jolokia/jolokia/blob/08902b1fb84d5c1e4e48154fee9c62f59b93ac9b/client/javascript/src/main/javascript/index.d.ts#L512-L516

Looking at the jolokia.js definition, `error_type` and `stacktrace` are also a required field. We should also change the jolokia.js type accordingly.

Reply:
I thought `ErrorResponse` is for entire response - it shouldn't be reused in nested, successful list response...